client/signup/steps/site-info-collection/form-components.tsx

```diff
@@ -0,0 +1,207 @@
+	return (
+		<FormFieldset>
+			<Label htmlFor={ props.name }>{ props.label }</Label>
+			<TextInput { ...props } isError={ false === props.isValid } />
+		</FormFieldset>
+	);
+}
+
+export function TextAreaField( props: TextInputFieldProps ) {
+	return (
+		<FormFieldset>
+			<Label htmlFor={ props.name }>{ props.label }</Label>
+			<TextArea
+				{ ...props }
+				isError={ false === props.isValid }
+				autoCapitalize="off"
+				autoCorrect="off"
+				spellCheck="false"
+			/>
+		</FormFieldset>
+	);
+}
+
+interface SocialMediaProfilesProps {
+	facebookProps: TextInputFieldProps;
+	twitterProps: TextInputFieldProps;
+	linkedinProps: TextInputFieldProps;
+	instagramProps: TextInputFieldProps;
+	onChange: ( event: ChangeEvent< HTMLInputElement > ) => void;
+}
+
+export function SocialMediaProfiles( {
+	facebookProps,
+	twitterProps,
+	linkedinProps,
+	instagramProps,
+	onChange,
+}: SocialMediaProfilesProps ) {
+	const translate = useTranslate();
+
+	const twitterLabel = (
+		<SocialMediaLabel>
+			<SocialLogo size={ 16 } fill="#8C8F94" icon="twitter" />
+			{ translate( 'Twitter' ) }
+		</SocialMediaLabel>
+	);
+	const facebookLabel = (
+		<SocialMediaLabel>
+			<SocialLogo size={ 16 } fill="#8C8F94" icon="facebook" />
+			{ translate( 'Facebook' ) }
+		</SocialMediaLabel>
+	);
+	const instagramLabel = (
+		<SocialMediaLabel>
+			<SocialLogo size={ 16 } fill="#8C8F94" icon="instagram" />
+			{ translate( 'Instagram' ) }
+		</SocialMediaLabel>
+	);
+	const linkedinLabel = (
+		<SocialMediaLabel>
+			<SocialLogo size={ 16 } fill="#8C8F94" icon="linkedin" />
+			{ translate( 'Linkedin' ) }
+		</SocialMediaLabel>
+	);
+	return (
+		<div>
+			<Label>
+				{ translate( 'Please enter the following Social Media profile links if you have any.' ) }
+			</Label>
+			<FlexContainer>
+				<TextInputField
+					{ ...twitterProps }
+					label={ twitterLabel }
+					placeholder="https://"
+					onChange={ onChange }
+				/>
+				<TextInputField
+					{ ...facebookProps }
+					label={ facebookLabel }
+					placeholder="https://"
+					onChange={ onChange }
+				/>
+				<TextInputField
+					{ ...instagramProps }
+					label={ instagramLabel }
+					placeholder="https://"
+					onChange={ onChange }
+				/>
+				<TextInputField
+					{ ...linkedinProps }
+					label={ linkedinLabel }
+					placeholder="https://"
+					onChange={ onChange }
+				/>
+			</FlexContainer>
+		</div>
+	);
+}
+
+interface ContactInformationProps {
+	displayEmailProps: TextInputFieldProps;
+	displayPhoneProps: TextInputFieldProps;
+	displayAddressProps: TextInputFieldProps;
+	onChange: ( event: ChangeEvent< HTMLInputElement > ) => void;
+}
+
+export function ContactInformation( {
+	displayEmailProps,
+	displayPhoneProps,
+	displayAddressProps,
+	onChange,
+}: ContactInformationProps ) {
+	const translate = useTranslate();
+	return (
+		<FlexContainer>
+			<TextInputField
+				{ ...displayEmailProps }
+				label={ translate( 'E-mail address' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **71** times:
`translate( 'Email address.' )` *ES Score: 6*